Base/Crypto/Crypto.Static/CryptoPrimitives.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+namespace Base::Crypto::Primitives
+{
+struct RSAKeyPair
+{
+    CryptoPP::RSA::PublicKey publicKey;
+    CryptoPP::RSA::PrivateKey privateKey;
```

Comment:
Is it cool to store private key in public section? Idk where this object stores in the other class, but it's still confusing to me rn

Reply:
'Private' means that you shouldn't publish this key. This struct is stored as private field of RSAKeyManager.